tests/screenshot/rgb.slint

```diff
@@ -0,0 +1,35 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+TestCase := Window {
+    width: 100px;
+    height: 100px;
+
+    GridLayout {
+        Rectangle { background: red; }
+        Rectangle { background: green; }
+        Rectangle { background: blue; }
+    }
+}
+   
+/*
+
+```rust
```

Comment:
This should be in the test driver itself.

Reply:
done